third_party/proton/dialect/lib/ProtonToProtonGPU/ProtonToProtonGPUPass.cpp

```diff
@@ -139,21 +169,32 @@ class ConvertProtonToProtonGPUPass
     }
 
     Value buffer;
+    auto ctaLayout =
+        triton::gpu::CTALayoutAttr::get(context, /*CTAsPerCGA=*/{1},
+                                        /*CTASplitNum=*/{1}, /*CTAOrder=*/{0});
+
+    // TODO(fywkevin): add another shared encoding attribute derived from the
```

Comment:
We probably do not have to add another shared encoding type. I'm OK with `SwizzledSharedEncodingAttr`